### ISSUE TITLE
Desktop app

### ISSUE THREAD

**Issue by StanGirard** [StanGirard/quivr#88]
Desktop app of the webapp. 

Need to think about how we proceed when there is no internet -> Local LLMs and embedding or full API ?

**Comment by JACKHAHA363** [comment]
It might be interesting to see LLama.cpp for local model

**Comment by DanielusG** [comment]
> It might be interesting to see LLama.cpp for local model

I manually modified the code (I haven't forked or committed yet) and managed to make it work with llama.cpp, I used the gpt4all 13b model.

 It lends itself well to this kind of activity.

Unfortunately I have to say that the performance is not good, the model is not able to respond adequately as gpt3 would, and to receive an answer it takes a long time. Despite my computer having an rtx 2060 and 16 cores of ryzen 7 4800h. As soon as I have time I will upload a fork of the repository with llama support. 

But don't have much hope that it will work well. I think maybe a 66B model could go a little better
